### DISCUSSION TITLE
Refresh automatically  a page after webhook

### DISCUSSION THREAD

**Discussion by nascondini** [nocodb/nocodb#7338] [Q&A]
Hi everyone.
How can I automatically update the page after the webooks event?
Is it possible to do this using the API?
Thank you!

**Comment by wingkwong** [comment]
which page you want to update and what is your use case?

**Comment by nascondini** [comment]
Hi, thanks for replying to me.
I'm sorry for my english.
When I enter new data, the webook activates and sends the data which is processed by n8n. Once processed, n8n updates the records via API. I need an automatic refresh of the iframe. Is there any way?
Thank you

**Reply by wingkwong** [reply]
the iframe meaning the share view from NocoDB?

**Reply by nascondini** [reply]
this 
![Hero-Image-p-2000](https://github.com/nocodb/nocodb/assets/139077045/243ae60a-f1f3-4642-b918-37a9ef8e3e23)
